src/admesh.c

```diff
@@ -450,7 +456,8 @@ redistribute it under certain conditions.  See the file COPYING for details.\n")
   }
 
   stl_stats_out(&stl_in, stdout, input_file);
-
+  if(fail_if_invalid_flag && stl_check_results(&stl_in, reverse_all_flag))
```

Comment:
If we do the reverse_all_flag if test outside of stl_check_results, then the function is more reusable outside of this main. WDYT?

Somethign like:

```suggestion
  if(fail_if_invalid_flag && !reverse_all_flag && stl_check_results(&stl_in))
```

Reply:
The reason is we still want to check other stats for validity even if we explicitly reverse facets. So (just reporting our IRL discussion) we will pass a number of facets that should be reversed and check that instead.